web-ui/src/helpers/checks.js

```diff
@@ -21,3 +21,48 @@ export function filterObjectByValOrKey(arr, value, key) {
       : a => Object.keys(a).some(k => a[k] === value)
   );
 }
+
+/**
+ * Sort Search Results for team members by Skill, Level & Name
+ * @param searchResults - an array of team members
+ * @returns a sorted array
+ */
+export function sortMembersBySkill(searchResults) {
+  const skillLevelsOrder = [
+    'EXPERT',
+    'ADVANCED',
+    'INTERMEDIATE',
+    'NOVICE',
+    'INTERESTED'
+  ];
+
+  if (isArrayPresent(searchResults)) {
+    // Sort the array by the number of skills
```

Comment:
This comment could be deleted since the two ways sorting is done are described in comments below.